invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordVersionsList.js

```diff
@@ -40,11 +40,12 @@ const RecordVersionItem = ({ item, activeVersion }) => {
         )}
 
         {doi && (
-          <small
+          <a
+            href={`https://doi.org/${doi}`}
```

Comment:
very minor: for some reason I prefer the underline always being visible. I think it's because "Version" above is already a link, so one could think they both link to the same destination, but they don't necessarily/are different links, so a visual distinctness helps. But this is perhaps fuzzy, so not a blocker.